packages/core/integration-tests/test/output-formats.js

```diff
@@ -695,7 +695,33 @@ describe('output formats', function() {
       );
     });
 
-    it('should generating ESM from CommonJS', async function() {
+    it.only('should create correct bundle import for reexports', async function() {
```

Comment:
Should we remove the `only` to execute all tests again?

Reply:
😬